app/javascript/components/rooms/room/room_settings/RoomSettings.jsx

```diff
@@ -66,6 +69,15 @@ export default function RoomSettings() {
                 config={roomConfigs?.glModeratorAccessCode}
                 description={t('room.settings.generate_mods_access_code')}
               />
+              {(!serverTags.isLoading && Object.keys(serverTags?.data).length !== 0) && (
```

Comment:
I don't think we need to explicitly handle the `isLoading` here. You can just do 
`const { data: serverTags } = useServerTags(friendlyId)` and then I would just check if `serverTags?.data?.length` and that should work

Reply:
True, I introduced the isLoading check when the second condition was not used yet.

Reply:
addressed in https://github.com/bigbluebutton/greenlight/pull/5790/commits/2919f08a6c7e030655af7dba104048d5dc5eef49

Reply:
@farhatahmad I thought I had a working solution without isLoading, but it turns out it can still happen I get errors like "can't convert undefined to object".

Reply:
resolved in 892385937b7a91d8c48bed0b9e9ab24280304acc